src/hooks/mixpanel.tsx

```diff
@@ -0,0 +1,49 @@
+import mixpanel from 'mixpanel-browser'
```

Comment:
any specific reason to define mixpanel here again ? why not use directly from components ?